Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.UI.ts

```diff
@@ -234,11 +235,7 @@ namespace Remotion.BocAutoCompleteReferenceValue
             serviceMethodSearch: serviceMethodSearch,
             serviceMethodSearchExact: serviceMethodSearchExact,
             data: null,
-            combobox: null,
-            selectListID: null,
-            informationPopUpID: null,
             // re-motion: clicking this control will display the dropdown list with an assumed input of '' (regardless of textbox value)
-            dropDownButtonID: null
```

Comment:
Why did you remove those?
And if removeing it is sensible, why leave the comment?

Reply:
Whoops forgot about the comment. I had TypeScript compilation problems because the properties are actually required but were registered as optional here. No idea why this has become a problem just now.